specs/latest/2.0/index.html

```diff
@@ -2120,8 +2140,9 @@ <h2 class="no-toc">Table of contents</h2>
               <tr><td>TEXTURE_WRAP_S</td><td>GLenum</td></tr>
               <tr><td>TEXTURE_WRAP_T</td><td>GLenum</td></tr>
           </table>
-          <p>If <em>pname</em> is not in the table above, generates an <code>INVALID_ENUM</code> error.</p>
-          <p>If an OpenGL error is generated, returns null.</p>
+          <p>If <em>pname</em> is not in the table above, generates an <code>INVALID_ENUM</code> error and returns null.</p>
```

Comment:
Agreed. Just referred other `get*` API like `getSyncParameter` and `getQueryParameter`. Maybe we can clean up them in another PR for consistency?